ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -218,17 +218,86 @@ class SignalProducerSpec: QuickSpec {
 			}
 		}
 
+		describe("ignoreValues") {
+			it("should not forward values from original producer") {
+				let producer: SignalProducer<Int, NoError> = SignalProducer<Int, NoError>(values: [ 1, 2 ]) |> ignoreValues
```

Comment:
Yep, nevermind. I was just in the scope of current test. Btw, I still think changing it may be good idea, since it doesn't really introduces new type.